### DISCUSSION TITLE
Tropical Moist Forest Transition Maps

### DISCUSSION THREAD

**Discussion by goergen95** [mapme-initiative/mapme.biodiversity#398] [Ideas]
# Tropical Moist Forest Transition Maps

## Definition of research question

Which kind of transitions and there areal share are observed within a PA?

## Definition of the indicator

As a starting point, the indicator simply reports annual transition statistics for 
tropical moist forests within a PA. More detailed analysis is likely to follow at a 
later stage.

The dataset was processed by the JRC to more clearly analyse forest cover transitions
than what is possible with the GFW layers. Starting from a mono-temporal analysis
of Landsat imagery between 1990 and today, pixels are classified as either with 
or without forest cover. In a following stage, based on the temporal dynamic of a pixel,
several classes are distinguished mainly indicating forest degradation, forest regrowth and
deforestation. 

The data set is distributed as tiles for each year in the time period under study. 
Since the observation starts only at 1990, the map cannot distinguish between primary forests
and areas where forests where observed at the beginning at the analysis but do not represent
primary forests. During the latest three years, the system cannot yet decide if it observed
degradation or deforestation, thus the observed changes are flagged as recent degradation/deforestation.

In total, there are 10 classes for the yearly raster layers:

- short-duration degradation followed by a recovery period (not followed by deforestation)
- long-duration degradation followed by a recovery period (not followed by deforestation)
- degradation followed by a recovery period, followed by deforestation
- degradation followed by a recovery period, followed by deforestation, and followed by 
  a recovery perion in itself (regrowth)
- deforestation of undisturbed forest, not followed by a recovery period
- deforestation of undisturbed forest, followed by a recovery period (forest regrowth)
- deforestation of a degraded forest (with a recovery period between degradation and deforestation stages)
- deforestation of a degraded forest (with a recovery period between degradation and deforestation stages), followed by a recovery period 
- deforestation with a forest conversion to water bodies
- deforestation with a forest conversion to commodities

where:
- short-duration degradation: observed within 1 year (e.g. logging, natural fires...)
- long-duration degradataion: observed within 1 to 2.5 years (e.g. strong fires)
- deforestation: distburbance events observed over periods > 2.5 years
- regrowth/recovery: a minimum of 3 years of permanent forest cover after deforestation
- commodities: plantation, agriculture, infrastructure, and others

## Possible data-sources

- [Web-Viewer](https://forobs.jrc.ec.europa.eu/TMF/explorer.php) 
- [Peer-Review Publication](https://www.science.org/doi/10.1126/sciadv.abe1603)
- [Downloads](https://forobs.jrc.ec.europa.eu/TMF/data.php#downloads)

## Spatio-temporal resolution

The annual change maps have a spatial resolution of 30 meters. They are distributed in 10°x10° tiles.
The spatial extent is restricted to the (sub-)tropics. The data is updated to they year 2022.

## Accessibility

The raster data set as well as other layers can be downloaded [here](https://forobs.jrc.ec.europa.eu/TMF/data.php#downloads).

**Comment by fBedecarrats** [comment]
Thank you for the framing @goergen95 ! Thanks also for the link to the viewer, which I didn't know about. One key aspect that strikes me when looking at the viewer is, besides the spatial extent of the dataset, the importance of the M (for "Moist") in T*M*F. This is documented in their Science publication, but I find it more appealing with pictures. An illustration with 3 regions:
**Amazon: "modest" differerences:**

- **GFC dataset:**
![Amazon_GFC](https://github.com/mapme-initiative/mapme.biodiversity/assets/3328347/6edd787f-b780-4e03-ace6-f9bf4a38f663)

- **TMF dataset:**
![Amazon_TMF](https://github.com/mapme-initiative/mapme.biodiversity/assets/3328347/4a2a3808-3030-4049-981d-640dc3479a93)

**Congo basin: "modest" differerences:**

- **GFC dataset:**
![Bongo_Basin_GFC](https://github.com/mapme-initiative/mapme.biodiversity/assets/3328347/1afeb3ab-b88c-44bd-aa96-c4182e6655f6)

- **TMF dataset:**
 ![Bongo_Basin_TMF](https://github.com/mapme-initiative/mapme.biodiversity/assets/3328347/4c58e42f-00b0-43c1-a83f-87e7a76486f9)

**Madagascar: "massive"

 differerences:**

- **GFC dataset:**
![Madagascar_GFC](https://github.com/mapme-initiative/mapme.biodiversity/assets/3328347/025f649b-d993-4338-8ef8-46d06fbb06ce)

- **TMF 
![Madagascar_TMF](https://github.com/mapme-initiative/mapme.biodiversity/assets/3328347/dae03135-4357-435c-8e96-55bc85f7ce51)
dataset:**

My impression from the above is that we should really keep in mind that this dataset monitors, better than the others, a specific type of forests, which are dominant in the Amazon, the Congo Basin and South East Asia, but that are only a minority of forest area in other regions, such as West Africa or Madagascar for instance. I'll try to bring in @MarcBouvier who works a lot on these datasets.

**Comment by Jo-Schie** [comment]
Hi there. So thanks for the comprehensive overview and also for highlighting the differences between both datasets. Since TMF is really specific to one (certainly important) forest type I became doubts again whether this should be prioritized. I guess it is also not possible to adjust the "forest definition" as in global forest watch. 

If we think of it from the perspective of analyzing protected areas it becomes a bit unclear to me if this data is really useful because we never now about the true extend of loss, degradation, gain etc. If not all possible forest types are covered. We might always tend to underestimate the true numbers with the problem become more severe as soon as we move further away from the equator. 

I guess it is interesting to look at the data on a map or study the subject itself (tropical moist forests) but the use value is now somehow limited. Not sure what you think @melvinhlwong .
